kolibri/plugins/user/assets/src/views/index.vue

```diff
@@ -1,12 +1,16 @@
 <template>
 
   <div>
-    <core-base :navBarNeeded="navBarNeeded" :topLevelPageName="topLevelPageName" :appBarTitle="appBarTitle">
-      <component v-if="navBarNeeded" :is="currentPage"/>
+    <core-base
```

Comment:
Having `core-base` optionally rendered will break browser page title functionality.

Need to either reinstate it, or extract the page title functionality from core-base to a mixin and apply here in a special case (while also continuing to include it in core-base).